devcheck/verifyContent.py

```diff
@@ -0,0 +1,41 @@
+import mmap, os
+from glob import glob
+
+# Verify that all resources contain an import.sh file and an entry in CHANGELOG.md
+has_no_import_files = False
+in_changelog = False
+no_import_dirs = []
+missing_changelog_entries = []
+
+dirs = glob("./examples/resources/*/", recursive=True)
+for dir in dirs:
+  resource_name = str(dir).split("/")[-2].encode()
+  with open(r'./CHANGELOG.md', 'rb', 0) as file:
+    s = mmap.mmap(file.fileno(), 0, access=mmap.ACCESS_READ)
+    if s.find(resource_name) != -1:
+      in_changelog = True
+      continue
+    else:
+      in_changelog = False
+      if resource_name not in missing_changelog_entries:
```

Comment:
You could make missing_changelog_entries a set so you don't have to check before appending